platform/core/src/services/ViewportGridService/ViewportGridService.ts

```diff
@@ -75,18 +71,8 @@ class ViewportGridService extends PubSubService {
     return this.serviceImplementation._getState();
   }
 
-  public setDisplaySetsForViewport({
-    viewportIndex,
-    displaySetInstanceUIDs,
-    viewportOptions,
-    displaySetOptions,
-  }) {
-    this.serviceImplementation._setDisplaySetsForViewport({
-      viewportIndex,
-      displaySetInstanceUIDs,
-      viewportOptions,
-      displaySetOptions,
-    });
+  public setDisplaySetsForViewport(viewport) {
+    this.serviceImplementation._setDisplaySetsForViewports([viewport]);
```

Comment:
this doesn't makes sense, the function is called `setDisplaySetsForViewport ` which should (as it sounds like), set some displaySet props and viewport props and set it, right now it is setting just viewport?

Reply:
What do you want me to call viewport when I set viewport?  I don't really care what it is called, but everywhere else we call it viewport, so I'd like to have a consistent new name for it if we are going to rename it.

Reply:
can't you just use the props? like the object that we already had?